### PR TITLE
High mem1

### PR DESCRIPTION
**Pull request by Slugger70**

### CODE REVIEW

File: files/galaxy/dynamic_job_rules/pawsey/dynamic_rules/destination_mapper.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
     'pulsar_mel2_user@usegalaxy.org.au': 'pulsar-mel_small',
     'pulsar_mel3_user@usegalaxy.org.au': 'pulsar-mel3_small',
     'pulsar_paw_user@usegalaxy.org.au': 'pulsar-paw_small',
+    'phm1@gneome.edu.au': 'pulsar-high-mem1_big',
```
Comment (cat-bro) [inline files/galaxy/dynamic_job_rules/pawsey/dynamic_rules/destination_mapper.py:17]: phm1@genome.edu.au

File: host_vars/pulsar-high-mem1/pulsar-high-mem1.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ssl_email: "help@genome.edu.au"
+
+#host specific pulsar settings
+
+rabbitmq_password_galaxy_au: "{{ vault_rabbitmq_password_galaxy_hm1_pawsey }}"
+pulsar_queue_url: "pawsey-queue.usegalaxy.org.au"
+pulsar_rabbit_username: "galaxy_hm1"
+pulsar_rabbit_vhost: "/pulsar/galaxy_hm1"
+
+attached_volume_device: /dev/vdb
```
Comment (cat-bro) [inline host_vars/pulsar-high-mem1/pulsar-high-mem1.yml:20]: attached_volumes:
  - device: /dev/vdb
    path: /mnt
    fstype: ext4

Comment (Slugger70) [reply host_vars/pulsar-high-mem1/pulsar-high-mem1.yml:20]: Actually, I have to remove that section altogether as I had to construct an LVM for it instead. :( `/dev/mapper/vg1-lvol0 on /mnt type ext4 (rw,relatime)`

File: files/galaxy/dynamic_job_rules/pawsey/dynamic_rules/destination_mapper.py (hunk 1 above)
Comment (Slugger70) [reply files/galaxy/dynamic_job_rules/pawsey/dynamic_rules/destination_mapper.py:17]: thanks. I'll fix it now

**Review by cat-bro — APPROVED** [review]

**Pull request merged**